packages/serverless/src/+awsLambda/awsLambda.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { map } from 'rxjs/operators';
+import { HttpMiddlewareEffect } from '@marblejs/core';
+import { getHeaderContent } from '../serverProxy.helpers';
+
+export const awsApiGatewayMiddleware$ = (): HttpMiddlewareEffect => (req$) => req$.pipe(
```

Comment:
It extracts `context` and `event` data to the request (as proxy packs it to headers) - unfortunately I didn't see a way to type it correctly and I'm not sure if I could use a context per each request. It can be skipped if you don't expect to use any AWS Lambda-provided values.